test/tester.go

```diff
@@ -244,6 +251,9 @@ func (at *AccountsTester) request(method string, endpoint string, queryParams ur
 	if at.Cookie != nil {
 		req.Header.Set("Cookie", at.Cookie.String())
 	}
+	if at.Token != "" {
+		req.Header.Set("Authorization", "Bearer "+at.Token)
+	}
 	client := http.Client{}
 	r, err := client.Do(req)
```

Comment:
The fact that we updated the file in 3 different spots with this `if` statement makes me think we should probably rather figure out a way to deduplicate all that http code.

Reply:
Done.